third_party/xla_client/BUILD

```diff
@@ -77,70 +95,72 @@ cc_library(
         ":unique",
         ":util",
         ":xla_util",
-        ":xrt_local_service",
-        ":xrt_session",
-        ":xrt_session_cache",
-        "@org_tensorflow//tensorflow:grpc++",
-        "@org_tensorflow//tensorflow/cc:client_session",
-        "@org_tensorflow//tensorflow/cc:scope",
-        "@org_tensorflow//tensorflow/compiler/jit:xla_cpu_device",
```

Comment:
looks like none of `jit` lines are support. can you confirm this line is supported @ManfeiBai?

cc @JackCaoG